rts/Net/GameServer.cpp

```diff
@@ -57,19 +57,29 @@
 #include "System/Platform/Threading.h"
 #include "System/Threading/SpringThreading.h"
 
+#include "System/Misc/TracyDefs.h"
+#include "System/TimeProfiler.h"
+
 #ifndef DEDICATED
 #include "lib/luasocket/src/restrictions.h"
 #endif
 
 #define ALLOW_DEMO_GODMODE
 
+//#define DEBUG_SERVERSYNCGAMETIMING // uncomment to enable debug output for serverSyncGameTiming
+#ifdef DEBUG_SERVERSYNCGAMETIMING
+	const char* const tracyFrameTimeLeft = "FrameTimeLeft";
+	const char* const tracyDrawSimRatio = "DrawSimRatio2";
+#endif
+
 using netcode::RawPacket;
 
 
 CONFIG(int, AutohostPort).defaultValue(0).description("Which port should the engine listen on for Autohost interfact connections.");
 CONFIG(int, ServerSleepTime).defaultValue(5).description("Number of milliseconds to sleep per tick for the server thread. Lower values have marginally higher CPU load, while high values can introduce additional latency.");
 CONFIG(int, SpeedControl).defaultValue(1).minimumValue(1).maximumValue(2)
 	.description("Sets how server adjusts speed according to player's load (CPU), 1: use average, 2: use highest");
+CONFIG(int, ServerSyncGameTiming).defaultValue(0).minimumValue(1).maximumValue(33).description("When non-zero Makes the server thread attempt to sync up simframe generation timing in local and demo games with the Games simframe consumption timing. Issues frames ~[ServerSyncGameTiming]ms before consumption, 3ms recommended");
```

Comment:
Is the 3ms empirically determined? The 33 max should probably be `1000/GAME_SPEED`.

Reply:
Yes, empirically determined means exactly what you think: I tested a lot of configurations and this gave the best results.